modular_skyrat/modules/container_emotes/code/container_emotes.dm

```diff
@@ -13,6 +13,8 @@
 	key_third_person = "exme"
 	message = null
 
+#define EXME_MAX_LOC_RECURSION 80 //no infinite loops
```

Comment:
could cut this down to like, 10, it's doubtful it'd even hit that

Reply:
true, but does it really hurt to have it this high?

Reply:
Yes, in a case of genuine recursion